src/features/history/History.tsx

```diff
@@ -13,6 +18,6 @@ export const History = () => (
       <Text style={styles.notificationTitle}>No history yet</Text>
       <Text style={styles.notificationInfo}>Hit the orange button down below to Create an order</Text>
     </View>
-    <CustomButton title="Complete order" color="#F6F6F9" backgroundColor="#FA4A0C" onPress={() => console.log(123)} />
+    <CustomButton title="Complete order" color="#F6F6F9" backgroundColor="#FA4A0C" onPress={() => MyToast.onCompletedOrders()} />
```

Comment:
@DiamondRock597 You can pass onCompletedOrders as callback, no need arrow function.